src/taskProvider.ts

```diff
@@ -107,6 +107,14 @@ export class TaskProvider implements vscode.TaskProvider {
       toolTip: "Run Python functional tests (with Qt display enabled)",
       enabled: true,
     },
+    {
+      group: "Functional Tests",
+      name: "Run tests with option 'golden_display'",
+      builders: { ["Both"]: this.functionalTestsGoldenRunExec },
+      dependsOn: this.functionalTestsRequirementsExec,
+      toolTip: "Run Python functional tests while storing the generated png files",
```

Comment:
"Run Python functional tests while storing the generated png files" 

I'd reword to 

"Run Python functional tests with --golden_run option to generate golden snapshots. They are used during tests runs to check what should be displayed on the device screen"

Reply:
ok